pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/StarTreeClusterIntegrationTest.java

```diff
@@ -135,14 +140,15 @@ public void setUp()
     List<File> avroFiles = unpackAvroData(_tempDir);
 
     // Create and upload segments
-    ClusterIntegrationTestUtils.buildSegmentsFromAvro(avroFiles, defaultTableConfig, schema, 0, _segmentDir, _tarDir);
-    uploadSegments(DEFAULT_TABLE_NAME, _tarDir);
-    TestUtils.ensureDirectoriesExistAndEmpty(_segmentDir, _tarDir);
-    ClusterIntegrationTestUtils.buildSegmentsFromAvro(avroFiles, starTreeTableConfig, schema, 0, _segmentDir, _tarDir);
-    uploadSegments(STAR_TREE_TABLE_NAME, _tarDir);
+    ClusterIntegrationTestUtils
+        .buildSegmentsFromAvro(avroFiles, defaultTableConfig, schema, 0, defaultTableSegmentDir, defaultTableTarDir);
+    uploadSegments(DEFAULT_TABLE_NAME, defaultTableTarDir);
+    ClusterIntegrationTestUtils
+        .buildSegmentsFromAvro(avroFiles, starTreeTableConfig, schema, 0, starTreeTableSegmentDir, starTreeTableTarDir);
+    uploadSegments(STAR_TREE_TABLE_NAME, starTreeTableTarDir);
 
     // Set up the query generators
-    SegmentInfoProvider segmentInfoProvider = new SegmentInfoProvider(_tarDir.getPath());
```

Comment:
shall this be `starTreeTableTarDir` or it doesn't matter?

Reply:
It doesn't matter